pkg/build/api/types.go

```diff
@@ -9,25 +9,32 @@ import (
 )
 
 const (
-	// BuildAnnotation is an annotation that identifies a Pod as being for a Build
+	// BuildAnnotation is an annotation that identifies a Pod as being for a
```

Comment:
remove these changes when you will be ready... they are unrelated to this PR and your name will appear in `git blame` ;-)